pkg/kubectl/cmd/apply.go

```diff
@@ -73,7 +73,7 @@ const (
 
 var (
 	apply_long = templates.LongDesc(i18n.T(`
-		Apply a configuration to a resource by filename or stdin.
+		Apply a configuration to the specified resource by filename or stdin with the exact matching resource name.
```

Comment:
@liggitt  Thanks for your suggestion. I think `descriptor` is a little not clear and may confuse the users. What about removing it like:
```
Apply a configuration to a resource by filename or stdin.
The resource name must be specified. This resource will be created if it doesn't exist yet.
To use 'apply', always create the resource initially with either 'apply' or 'create --save-config'.
```